engine/src/cython/initialize.cpp

```diff
@@ -274,27 +292,15 @@ void finalize() {
 	exit(0);
 }
 
-void blazingSetAllocator(
```

Comment:
where is this api now?

Reply:
It is now in the singular initialize API